tests/deprecations_ignore.yml

```diff
@@ -70,9 +70,6 @@
 
 
 # Core
-- tests/core/test_core.py::TestCore::test_dag_params_and_task_params
-- tests/core/test_core.py::TestCore::test_externally_triggered_dagrun
-- tests/core/test_impersonation_tests.py::TestImpersonation::test_impersonation_subdag
```

Comment:
This one executed as Quarantined test case, this is not false negative

```console
pytest tests/core/test_impersonation_tests.py::TestImpersonation::test_impersonation_subdag --include-quarantined

...

ERROR [airflow.models.dagbag.DagBag] Failed to import: /opt/airflow/tests/dags/test_impersonation_subdag.py
Traceback (most recent call last):
  File "/opt/airflow/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/tests/dags/test_impersonation_subdag.py", line 49, in <module>
    subdag_operator = SubDagOperator(
  File "/opt/airflow/airflow/models/baseoperator.py", line 489, in apply_defaults
    result = func(self, **kwargs, default_args=default_args)
  File "/opt/airflow/airflow/utils/session.py", line 79, in wrapper
    return func(*args, session=session, **kwargs)
  File "/opt/airflow/airflow/operators/subdag.py", line 100, in __init__
    warnings.warn(
airflow.exceptions.RemovedInAirflow3Warning: This class is deprecated. Please use `airflow.utils.task_group.TaskGroup`.
-------------------------------------------------- Captured log setup ---------------------------------------------------
INFO     airflow.utils.db:db.py:1682 Dropping tables that exist
INFO     alembic.runtime.migration:migration.py:216 Context impl PostgresqlImpl.
INFO     alembic.runtime.migration:migration.py:219 Will assume transactional DDL.
INFO     alembic.runtime.migration:migration.py:216 Context impl PostgresqlImpl.
INFO     alembic.runtime.migration:migration.py:219 Will assume transactional DDL.
----------------------------------------------- Captured stderr teardown ------------------------------------------------
userdel: airflow_test_user mail spool (/var/mail/airflow_test_user) not found
================================================ short test summary info ================================================
FAILED tests/core/test_impersonation_tests.py::TestImpersonation::test_impersonation_subdag - AttributeError: 'NoneType' object has no attribute 'clear'
============================================= 1 failed, 1 warning in 6.68s ==============================================
```

Reply:
Yes you are right. Let me add it back for now.